hackathon_site/registration/jinja2/registration/emails/accepted_email_body.html

```diff
@@ -0,0 +1,10 @@
+<p>Hello {{ user.first_name|striptags }},</p>
+
+<p>The {{ hackathon_name }} team has reviewed your application, and we’re excited to welcome you to {{hackathon_name}} 2020! 
+To confirm your attendance, visit your <a href="https://ieee.utoronto.ca/makeuoft/dashboard">dashboard</a> to RSVP by {{rsvp_deadline}} (11:59 PM EST) otherwise your spot will be given to other applicants. Alternatively, you can press the “Cannot attend” button.</p>
```

Comment:
The use of smart quotes `“Cannot attend”` here scares me, I assume they came from copying from the doc. Can these be replaced with regular (dumb) double quotes?

Reply:
The 11:59 PM EST doesn't need to be in the template, we can put that in `rsvp_deadline` when using `.strftime()` since it's computed from a datetime object